hadoop-plugin/src/main/groovy/com/linkedin/gradle/azkaban/AzkabanHelper.groovy

```diff
@@ -144,7 +143,6 @@ class AzkabanHelper {
    * @return The parsed response
    */
   static String parseResponse(String response) {
-    logger.lifecycle("AzkabanHelper: parseResponse");
     String newline = System.getProperty("line.separator");
     return HtmlUtil.toText(response).replaceAll("azkaban.failure.message=", newline + newline);
```

Comment:
Fixed it in LiAzkabanPlugin.groovy
azkabanProject.azkabanUrl = "https://ltx1-holdemaz01.grid.linkedin.com:8443"; where the trailing slash was at the end of the Url